test/system/libs/helpers.bash

```diff
@@ -577,3 +577,56 @@ function is_fedora_rawhide() (
 
   return 0
 )
+
+
+# Creates a container with org.freedesktop.Flatpak.SessionHelper D-Bus interface
+#
+# Pulling of an image is taken care of by the function
+#
+# Parameters:
+# ===========
+# - container_name - name of the container
+function create_container_flatpak_session_helper() (
+  local container_name="$1"
+
+  local default_image
+  default_image="$(get_default_image)"
+
+  pull_default_image
+
+  gdbus call \
+    --session \
+    --dest org.freedesktop.Flatpak \
+    --object-path /org/freedesktop/Flatpak/SessionHelper \
+    --method org.freedesktop.Flatpak.SessionHelper.RequestSession
+
+  podman_cmd=(
+      podman create \
+          --dns none \
+          --env TOOLBOX_PATH="$TOOLBX" \
+          --name "$container_name" \
+          --network host \
```

Comment:
It might be good to add the labels, because you have been working on checking the labels, and it will let us add another test to `test/system/102-list.bats` to ensure that these deprecated containers are listed.